src/views/AlumniUpdateForm/components/UpdateForm/index.js

```diff
@@ -0,0 +1,349 @@
+import {
+  Typography,
+  Grid,
+  Card,
+  CardContent,
+  CardHeader,
+  Button,
+  TextField,
+} from '@material-ui/core/';
+
+import { useState, useMemo, useEffect, useCallback } from 'react';
+import { useHistory } from 'react-router-dom';
+import { requestInfoUpdate, getAllStates, getAllCountries } from 'services/update';
+import styles from '../Update.module.css';
+import GordonLimitedAvailability from 'components/GordonLimitedAvailability';
+import GordonLoader from 'components/Loader';
+import SimpleSnackbar from 'components/Snackbar';
+import GordonOffline from 'components/GordonOffline';
+import useNetworkStatus from 'hooks/useNetworkStatus';
+import GordonDialogBox from 'components/GordonDialogBox';
+import { ConfirmationRow } from '../ConfirmationRow';
+import { ConfirmationWindowHeader } from '../ConfirmationHeader';
+import { ContentCard } from '../ContentCard';
+import { ProfileUpdateField } from '../ProfileUpdateField';
+
+const personalInfoFields = [
+  {
+    label: 'Salutation',
+    name: 'salutation',
+    type: 'select',
+    menuItems: ['Prefer Not to Answer', 'Mr.', 'Ms.', 'Mrs.', 'Miss', 'Dr.', 'Rev.'],
+  },
+  { label: 'First Name', name: 'firstName', type: 'text' },
+  { label: 'Last Name', name: 'lastName', type: 'text' },
+  { label: 'Middle Name', name: 'middleName', type: 'text' },
+  { label: 'Preferred Name', name: 'nickName', type: 'text' },
+  { label: 'Suffix', name: 'suffix', type: 'text' },
+  { label: 'Married', name: 'married', type: 'checkbox' },
+];
+const emailInfoFields = [
+  { label: 'Personal Email', name: 'personalEmail', type: 'email' },
+  { label: 'Work Email', name: 'workEmail', type: 'email' },
+  { label: 'Alternate Email', name: 'aEmail', type: 'email' },
+  {
+    label: 'Preferred Email',
+    name: 'preferredEmail',
+    type: 'select',
+    menuItems: ['No Preference', 'Personal Email', 'Work Email', 'Alternate Email'],
+  },
+];
+const phoneInfoFields = [
+  { label: 'Home Phone', name: 'homePhone', type: 'number' },
+  { label: 'Work Phone', name: 'workPhone', type: 'number' },
+  { label: 'Mobile Phone', name: 'mobilePhone', type: 'number' },
+  {
+    label: 'Preferred Phone',
+    name: 'preferredPhone',
+    type: 'select',
+    menuItems: ['No Preference', 'Home Phone', 'Work Phone', 'Mobile Phone'],
+  },
+];
+const shouldContactFields = [
+  { label: 'Do Not Contact', name: 'doNotContact', type: 'checkbox' },
+  { label: 'Do Not Mail', name: 'doNotMail', type: 'checkbox' },
+];
+
+/**
+ * A form for alumni to request an update to their profile information.
+ */
+
+const UpdateForm = (props) => {
+  const history = useHistory();
+  const returnToMyProfile = useCallback(() => history.push('/myprofile'), [history]);
+
+  const isOnline = useNetworkStatus();
+  const profile = props.profile;
```

Comment:
You can destructure props instead
```suggestion
const UpdateForm = ({ profile }) => {
  const history = useHistory();
  const returnToMyProfile = useCallback(() => history.push('/myprofile'), [history]);

  const isOnline = useNetworkStatus();
```